nengo_viz/static/viz_netgraph_item.js

```diff
@@ -120,11 +124,13 @@ VIZ.NetGraphItem = function(ng, info) {
 
     if (!this.passthrough) {
         /** dragging the edge of item to change its size */
-        var tmp = this.shape
-        if(info.type === 'ens') {
-            tmp = $(this.shape.getElementsByClassName('mainCircle'))[0];
+        var items = [this.area];
+        if (info.type !== 'ens') {
+            items.push(this.shape);
         }
-        interact(tmp)
+        for (var i = 0; i < items.length; i++) {
+          var tmp = items[i];
```

Comment:
:)  I agree -- that was just there because I was feeling lazy and was just trying out that for loop to see if it works.  Should definitely be fixed before merging, if we decide to go this route.